snooty/postprocess.py

```diff
@@ -578,6 +578,7 @@ def exit_node(self, fileid_stack: FileIdStack, node: n.Node) -> None:
 class TabsSelectorHandler(Handler):
     def __init__(self, context: Context) -> None:
         super().__init__(context)
+        self.default_tabs: Dict[Optional[str], Optional[str]] = {}
```

Comment:
Do both the key and value in the dictionary need to be an optional string? I think the key can safely be a `str` for sure. Do we ever want to support the value being`None`? The check for `"default-tabid" in node.options` seems to suggest we always want a string